Sources/AnimalKingdomAPI/AnimalKingdomAPI/Sources/Operations/AllAnimalsQuery.swift

```diff
@@ -428,11 +428,11 @@ public class AllAnimalsQuery: GraphQLQuery {
         public static var __parentType: ParentType { .Object(AnimalKingdomAPI.Dog.self) }
         public static var selections: [Selection] { [
           .field("favoriteToy", String.self),
-          .field("birthdate", AnimalKingdomAPI.CustomDate?.self),
```

Comment:
This namespace was removed because the configuration of the operation files in AnimalKingdom are `.inSchemaModule`.